crates/data_components/src/databricks/sql_warehouse.rs

```diff
@@ -163,14 +163,16 @@ impl SqlWarehouseApi {
 
     async fn get_schema(&self, table: &TableReference) -> Result<SchemaRef, Error> {
```

Comment:
Do we need to update some of the integration tests to include `describe table` snapshot to verify nullability is correct?